### DISCUSSION TITLE
Relative links in member only readme files for organization

### DISCUSSION THREAD

**Discussion by ahmed-shariff** [community/community#43516] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When I add relative links to the `.github-private/profile/Readme.md`, they point to different links depending on if the link is rendered on the organizations profile page or the repo itself.

I am assuming this is a bug, hence,  categorizing it as a bug.

**Comment by jfullstackdev** [comment]
the possibility is just like in VS Code, if you are developing using Modules, the current directory is where you put the link, so it will be a different relative link from time to time

so, in my case, I'm using absolute when I know, the address will not be changing from time to time
